workspace.json

```diff
@@ -140,11 +140,45 @@
       "projectType": "library",
       "implicitDependencies": []
     },
+    "@fluentui/nx-workspace-tools": {
```

Comment:
I only expect to see the change for `react-toolbar` here the rest of these are strange, can you please double check ?

Reply:
I have no idea how this was generated. Any idea?